lib/mongo.js

```diff
@@ -96,6 +96,16 @@ class Mongo {
     })
   }
 
+  async updateManuallyBilledRemainingDonation ({ organizationId, remainingDonation }) {
+    return this.db.collection(ORGS_COLLECTION).updateOne({
+      _id: ObjectId(organizationId)
+    }, {
+      $set: {
+        remainingDonation
+      }
+    })
+  }
```

Comment:
I think it would be "better" to use `$inc: { remainingDonation: -amount }`, since `$inc` is atomic. I'm not sure what kind of race conditions we might hit with `$set`, but it feels clean to me to use `$inc`.

Reply:
i agree, i actually looked for $dec and forgot we can just $inc negative values 😭